src/TestLogger/Core/XmlSanitizer.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Spekt Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Spekt.TestLogger.Core
+{
+    using System.Text.RegularExpressions;
+
+    internal static class XmlSanitizer
+    {
+        internal static string RemoveInvalidXmlChar(string str)
```

Comment:
From copying this in, I realized we don't have nullable enabled in this proj. Any reason not to add it?

Reply:
We can add it depending on use case.

Reply:
will leave it out for now